app/views/users/new.html.erb

```diff
@@ -0,0 +1,20 @@
+  <div class="col-md-4 col-md-offset-4">
+    <%@user.errors.full_messages.each do |message|%>
+      <div class="danger"><%= message%></div>
+    <%end%>
+
+  <h3 class="text-center">Register</h3>
+  <%= form_for @user do |f| %>
+    <%= f.label "Name:"%>
+    <%= f.text_field :username,class: 'form-control'%>
```

Comment:
- sadly i'm unable to login.
![Screenshot 2019-09-02 at 3 08 29 PM](https://user-images.githubusercontent.com/10922508/64120381-f244ea00-cd93-11e9-9fcc-96ae9831c47d.png)